arcon_allocator/Cargo.toml

```diff
@@ -4,6 +4,13 @@ version = "0.1.0"
 authors = ["Max Meldrum <mmeldrum@kth.se>"]
 edition = "2018"
 
+[features]
+allocator_metrics = ["metrics-exporter-prometheus", "metrics"]
```

Comment:
We don't need the prom exporter in the ``arcon_allocator`` crate so having metrics as optional dependency only is fine.  Which can be activated through ``arcon_allocator/metrics``

Reply:
yeah good point!